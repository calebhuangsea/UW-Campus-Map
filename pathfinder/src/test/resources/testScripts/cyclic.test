# test find path in a cyclic graph

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

AddEdge graph1 n1 n2 1.000
AddEdge graph1 n2 n3 1.000
AddEdge graph1 n3 n1 1.000

FindPath graph1 n1 n1