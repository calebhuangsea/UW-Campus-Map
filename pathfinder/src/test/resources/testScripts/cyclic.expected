# test find path in a cyclic graph

created graph graph1

added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

added edge 1.000 from n1 to n2 in graph1
added edge 1.000 from n2 to n3 in graph1
added edge 1.000 from n3 to n1 in graph1

path from n1 to n1:
total cost: 0.000